salt/utils/vt.py

```diff
@@ -414,10 +414,21 @@ def _spawn(self):
             else:
                 args = []
 
+            # Decide Which Shell To Use
+            shell_path = "/bin/sh"
+
+            # /bin/sh -> To Start
+            # /bin/bash -> For Most Linux
+            # /bin/ksh -> For AIX/HP-UX and the Like
+            # /usr/local/bin/tcsh -> For FreeBSDers
+            for best_shell in ["/bin/bash", "/bin/ksh", "/usr/local/bin/tcsh"]:
```

Comment:
tcsh is not in /usr/local/bin on FreeBSD.  It's in /bin because it is part of the default install.  However if you are looking for root's default shell it's `/bin/csh`.

Reply:
Okay I believe I fixed that. Sorry for the delay.